mmdeploy/codebase/mmdet/models/misc.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    func_name='mmdet.models.utils.misc.generate_coordinate')
+def generate_coordinate__onnx(ctx, featmap_sizes, device='cuda'):
+    x_range = torch.arange(
```

Comment:
The purpose of this rewriter is to replace linspace to arange, right?
Can we add rewriter for `torch.linspace`? in `mmdeploy/pytorch/functions ?